test/make-test.js

```diff
@@ -11,6 +11,7 @@ function makeTest (t, options, isOk, confExpected, errorMessage) {
     t.strictSame(conf, confExpected)
   } catch (err) {
     if (isOk) {
+      console.log(err)
```

Comment:
Why this `console.log`?

Reply:
That was me debugging the errors raised by the test. There are still some failures that I am trying to resolve.

Reply:
Removed now.